subprojects/docs/src/snippets/testing/jacoco/quickstart/groovy/build.gradle

```diff
@@ -48,6 +48,10 @@ test {
 }
 // end::testtask-configuration[]
 
+// tag::testtask-dependency[]
+test.finalizedBy jacocoTestReport // report is always generated after tests run
+jacocoTestReport.dependsOn test // tests are required to run before generating the report
+// end::testtask-dependency[]
```

Comment:
:o: Nit: maybe consider using blocks to configure the tasks in Groovy, just like in Kotlin, for similarity. This would also make both snippets to have the same height in the guide hence not causing layout jumps when switching.

```groovy
test {
    finalizedBy(jacocoTestReport) // report is always generated after tests run
}
jacocoTestReport {
    dependsOn(test) // tests are required to run before generating the report
}
```

Reply:
Good thinking, thanks.